include/lsst/jointcal/Gtransfo.h

```diff
@@ -179,6 +180,17 @@ std::ostream &operator<<(std::ostream &stream, Gtransfo const &gtransfo);
  */
 std::unique_ptr<Gtransfo> gtransfoCompose(Gtransfo const &left, Gtransfo const &right);
 
+/**
+ * Transform a bounding box, taking either the inscribed or circumscribed box.
+ *
+ * @param[in]  inputframe  The frame to be transformed.
+ * @param[in]  gtransfo    The transform to apply.
+ * @param[in]  which       inscribed (frame::SmallFrame) or circumscribed (frame::LargeFrame) box.
+ *
+ * @return     The transformed frame.
+ */
+Frame applyTransfoToFrame(Frame const &inputframe, Gtransfo const &gtransfo, WhichTransformed const which);
```

Comment:
Why isn't this an `apply` method on `Gtransfo`?

If `WhichTransformed` only has two possible values, why not make it a `bool`?